nixos/modules/security/acme.nix

```diff
@@ -136,6 +136,25 @@ let
           challenge to ensure the DNS entries required are available.
         '';
       };
+
+      ocspMustStaple = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Turns on the OCSP Must-Staple TLS extension.
+          Make sure you know what you're doing! See:
+          https://blog.apnic.net/2019/01/15/is-the-web-ready-for-ocsp-must-staple/
```

Comment:
Real nitpick here... but wrapping in a `link` tag or whatever is always nice for easy clicking!

Reply:
done, now it's unreadable in the text form instead :p

I would like to note that the equivalent AsciiDoc – corresponding to the exact same document tree and using the same document model – would be simply:

```asciidoc
* https://blog.apnic.net/2019/01/15/is-the-web-ready-for-ocsp-must-staple/
* https://blog.hboeck.de/archives/886-The-Problem-with-OCSP-Stapling-and-Must-Staple-and-why-Certificate-Revocation-is-still-broken.html
```

and register my confusion as to why we didn't switch to it long ago.

Reply:
@emilazy please see https://github.com/NixOS/rfcs/pull/64 for discussion. I would encourage you to participate in the RFC process to share your input!

Reply:
ok, left my 2¢ ^^